docs/steering-committee/.pages.yml

```diff
@@ -1,2 +1,3 @@
+---
```

Comment:
Please remove this file instead, it isn't used (I think).

Reply:
fixed 38411e46ed59e835bd55a8351bc3755265a420b3